hack/ci-deploy.sh

```diff
@@ -45,6 +45,7 @@ for f in $( find "${deploy_dir}"/ -type f -name '*.yaml' ); do
 done
 
 yq e --inplace '.spec.template.spec.containers[0].args += ["--qps=200", "--burst=400"]' "${deploy_dir}/operator/50_operator.deployment.yaml"
+yq e --inplace '.spec.template.spec.containers[0].args += ["--crypto-key-buffer-size-min=3", "--crypto-key-buffer-size-max=6", "--crypto-key-buffer-delay=2s"]' "${deploy_dir}/operator/50_operator.deployment.yaml"
```

Comment:
are default numbers causing any issues in CI?

Reply:
We are very limited on CPU cycles and we do not yet have limits on operator consumption (#1204) so as the cpu is shared for all processes any extra work will extend the CI run time at best, possibly times out the suite it individual waits in it. I've tried it when the default were higher and it didn't go well.